code/modules/mob/living/carbon/xenomorph/xeno_defines.dm

```diff
@@ -1052,7 +1054,7 @@
 	hugger.timeofdeath = user.timeofdeath // Keep old death time
 
 /datum/hive_status/proc/update_lesser_drone_limit()
-	lesser_drone_limit = Ceiling(totalXenos.len / 3)
+	lesser_drone_limit = lesser_drone_minimum + Ceiling(length(totalXenos) / 3)
```

Comment:
This math is bad and will lead to hives having +3 drones at any size, this needs to be like so (mobile, so shit formatting)

max(minimum drones, ceiling of total xenoes / 3)